tensorflow/tools/ci_build/install/install_pip_packages.sh

```diff
@@ -86,7 +86,8 @@ pip3 install lazy-object-proxy==1.4.3
 # pylint tests require the following version. pylint==1.6.4 hangs erratically,
 # thus using the updated version of 2.5.3 only for python3 as python2 is EOL
 # and this version is not available.
-pip3 install pylint==2.5.3
+pip3 install pylint==2.6.2
+pip3 install astroid==2.5
```

Comment:
Let's put these on the same line, here and elsewhere.

This would make pip install them in a single pass and use a common set of dependencies.

Reply:
That was the original idea but I don't know if it violated any policies :winkle: